### ISSUE TITLE
i really tried but failed ;(

### ISSUE THREAD

**Issue by mmollet** [pierrecle/Hackintosh-NUC9I7QNX-OpenCore#1]
Hi, i tried to replicate an installation with your files but it always fails after first setup step.
Ive flashed the bios you provided.
Ive facefully did the bios settings just as you mentioned and loaded defaults before i set them.
Ive created an SMBios
Ive used pythom macrecovery.py to download newest Big sur using:
macrecovery.py -b Mac-E43C1C25D4880AD6 -m 00000000000000000 -os latest download
ive formattet the usb driive with fat32.
created the folder, put the 2 files from osx download insight.
put the Efi folder with edited config.plist (platform info) on usb drive.
Bootet, cleaned the m2 (500GB) with Disk Managemet Utlitiy and formattet to journaled.
Hit Install OSX Big Sur.
Then after about 3/4 of the installation my Nuc reboots and i choose MaxOS Installer in Boot menu.
Then the Nuc reboots again and iam at the starting page where it says restore from timemachine, macOS Big sur resinstall.
Iam not geeting past this 3/4 of the installation.
USB wise i have connected a Corsair K100 and a Razer Deathadder Pro.
On HDMI i have connected an Asus 43Inch 4k Monitor.

I have not installed a extern GPU.
I have not installed a second M2.

The USB Drive is plugged in the fron USB of the Nuc.

My Nuc is an: Nuc9i9QNX1

Thank you very much for your work and the support to the hackintosh community.

**Comment by bemble** [comment]
Hi,

I think you're very close to your first boot!
Just a dummy question: in the OpenCore boot menu, do you have an option named by the name of your installation partition (for me "Mac OS")?
I also had multiple reboots during my installation, and after that, I had a new line in boot menu.

**Comment by mmollet** [comment]
Hi, thanks for replying.

In the Video you see what i mean.

https://drive.google.com/file/d/1y4s_ZNqxGDcQhjPUGXYYcShAIrKxIyrQ/view?usp=sharing

also this is the Screenshot of the debug after the first boot retry.

https://drive.google.com/file/d/16e4FiStAnMFeCNBGijXfHHGn1RwKL95g/view?usp=sharing

BTW: i already have multiple hackintoshs running on different platforms. But this EFI is not really working for me.

**Comment by bemble** [comment]
Thanks for the image, it helps. It seams like MacOS is trying to start with secure boot. Try to disable it: https://dortania.github.io/OpenCore-Post-Install/universal/security/applesecureboot.html#securebootmodel

**Comment by mmollet** [comment]
thanks again, i already tried that option earlier. Problem is, in opencore boot menu usb is still working but when i get past the "old" error and it says 29 minutes left i get this like 10 seconds later.

https://drive.google.com/file/d/1tq6bDAr7ZNyGgl6iXTJCW-GmSjG2q5bm/view?usp=sharing

and no usb is working anymore.

I also mentioned that you say untick USB legacy Support. In my Bios (the file from you) there are 3 options i NEED to choose one.
Enabled, Disabled, Auto. I tried all of them but still same error when setting SecureBoot to Disabled.

**Comment by bemble** [comment]
> and no usb is working anymore.

I changed `SetupBootModel`, after first reboot everything still working, another reboot and no more USB for me to, I try to fix it (rollback to `Default` doesn't make it work..), and I'll look closer to my BIOS config.

> I also mentioned that you say untick USB legacy Support. In my Bios (the file from you) there are 3 options i NEED to choose one.

You're right, it should be "Enabled", I fixed it.

**Comment by bemble** [comment]
New found: my USB issue is due to the latest Mac OS update (11.3 - 20E232)
Details about latest update here: https://www.tonymacx86.com/threads/macos-11-3-update.312150/

@mmollet So, because you're using latest version, it might be an issue related to the latest update. Could you try install using 11.2.3 - 20D91?

**Comment by mmollet** [comment]
thank you for investigating this. 

Biggest Problem now, I have no idea how to get on an "old" Version of big sur to install it.
Even when i have the base image i guess it would automatically download the updated files in the setup process...

**Comment by mmollet** [comment]
i finally made it ... ive used a complicated way to make an usb installer from a full version pkg download for the suggested version 20D91. I keep it like that and dont change anyhting. I rarly use maxos only for mobile device management and some other things, it should be good for that even its not the newest version. Thank you very very much for your help and your support ! i really like how detailed you made your readme !!! very well done !

**Comment by bemble** [comment]
> i finally made it ... ive used a complicated way to make an usb installer from a full version pkg download for the suggested version 20D91. I keep it like that and dont change anyhting. I rarly use maxos only for mobile device management and some other things, it should be good for that even its not the newest version.

🍾 Glad to read that! I used a Mac to create my install USB, with full version. It's like I miss something in OpenCore config to be able to install it from recovery, but I don't know what AND I didn't use this method so... I think I will preciously keep a full version of 20D91 somewhere, In case of emergency 😄 

> Thank you very very much for your help and your support ! i really like how detailed you made your readme !!! very well done !

Thanks, I try to keep all information I had to find here, to avoid frustration I had when spending hours trying to find information, mostly partial information. When I tried to install I mostly found forum posts saying "add this line in config.plist" or "add this patch", without any BIOS configuration information, kext version, or any information that could help a newbie like me. And I have to say, I do it for me first of all, if I have to reinstall everything in few months, I will be happy to find it.
